src/behaviors/animad-relation-behavior.html

```diff
@@ -2,7 +2,7 @@
 
 <script>
     /**
-     * Behavior, um einem Relation Element mit den notwendigen Eigenschaften 
```

Comment:
Woher kommen denn in dieser Datei die ganzen Änderungen?

Reply:
Hm, da sind Leerzeichen etc. rausgelöscht worden. Kann schon sein, dass ich das File aufgemacht hab, aber bewusst geändert hab ich da nichts. Entweder der Editor oder beim ?Merge??

Wenn du möchtest kann ich das rückgängig machen.